src/apps/schedule/api/schedule.py

```diff
@@ -132,7 +132,12 @@ async def schedule_get_all_by_user(
     schedules = await ScheduleService(session).get_events_by_user(
         user_id=user.id
     )
-    return ResponseMulti(result=schedules)
+    return ResponseMulti(
+        result=schedules,
+        count=sum(
+            [len(schedule.events) for schedule in schedules if schedule.events]
```

Comment:
pls make it query, because it there will be many schedules, it will consume time to calculate